scripts/polkadotjs_endpoints_to_preconfigured.py

```diff
@@ -275,16 +275,18 @@ def process_json_file(file_path):
 
 
 def process_dict_data(file_path, data):
-    if "(SHUTTING DOWN)" not in data.get('name', ''):
+    if "(SHUTTING DOWN)" in data.get('name', '') or "Westend" in data.get('name', ''):
```

Comment:
That wouldn't work properly, `"Westend" in data` will match with many networks, for example: "Westend BridgeHub (TESTNET)" or "Westend Collectives (TESTNET)"

I would suggest to add "Westend (TESTNET)" as a pattern here and move it to separate array, example:

```python
skip_patterns = ["(SHUTTING DOWN)", "Westend (TESTNET)"]

if any(pattern in data.get('name', '') for pattern in skip_patterns):
    pass
    print(f"Skipped file: {file_path}")
os.remove(file_path)
print(f"Removed file: {file_path}")
```